src/languages/TypeScript.ts

```diff
@@ -0,0 +1,21 @@
+import TreeSitterTypeScript from 'tree-sitter-typescript/typescript';
+import { findDeclaration } from './util';
+
+const NODE_TYPES = Object.freeze([
+    'class',
+    'class_declaration',
+    'function',
+    'function_declaration',
+    'generator_function',
+    'generator_function_declaration',
+    'method_definition',
```

Comment:
We should add `abstract_class_declaration`, `interface_declaration`, `enum_declaration`, `type_alias_declaration`, `property_signature` (what do you think about this one?), `module` to both language files.